cl/assets/templates/cotton/tabs/index.html

```diff
@@ -0,0 +1,59 @@
+{% load component_tags %}
+{% require_script "js/alpine/components/tabs.js" %}
+{% require_script "js/alpine/components/focus.js" %}
```

Comment:
the path for the `focus.js` file seems to be incorrect as well 😅 

```suggestion
{% require_script "js/alpine/composables/focus.js" %}
```